src/Destructurama.ByIgnoring/Destructurama.ByIgnoring.csproj

```diff
@@ -31,8 +31,9 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Condition="'$(TargetFramework)' == 'netstandard1.0'" Include="Serilog" Version="2.5.0" />
-    <PackageReference Condition="'$(TargetFramework)' == 'netstandard2.0'" Include="Serilog" Version="[2.8.0,3.0.0)" />
+    <PackageReference Condition="'$(TargetFramework)' == 'netstandard1.0'" Include="Serilog" Version="[2.5.0,3.0.0)" />
+    <PackageReference Condition="'$(TargetFramework)' == 'netstandard2.0'" Include="Serilog" Version="[2.8.0,4.0.0)" />
+    <PackageReference Condition="'$(TargetFramework)' == 'netstandard2.1'" Include="Serilog" Version="[2.8.0,4.0.0)" />
```

Comment:
I think we can/should drop the conditional dependency versioning here, things end up rather messy in multi-project solutions. Targeting Serilog 3.0.1 would be fine.

Reply:
Agree

Reply:
What about netstandard1.0? It requires a lower Serilog as 3.0.1 doesn't support it. I'll make the change for the netstandard2.0+...

Reply:
Here we go.  If I leave the package range in the solution will choose the one needed for the netstandard1.0. 👍

Reply:
Should I have left this unresolved for visibility?  I think I will unresolve this comment section.  I really could use the updated Serilog, but cannot because the current version of Destructurama.By-Ignoring requires < 3.0.

Reply:
Destructurama.ByIgnoring should definitely work with Serilog 3; what error are you seeing?

Reply:
NuGet downgrades Serilog to 2.12.0 because the Destructurama.By-Ignoring NuGet package has a [dependency](https://www.nuget.org/packages/Destructurama.ByIgnoring#dependencies-body-tab) limit of >= 2.8.0 && < 3.0.0.  I created this pull request to update the dependency to include Serilog 3.*.

Reply:
Thanks, I see the problem now 👍